### ISSUE TITLE
All page content should be contained by landmarks

### ISSUE THREAD

**Issue by yusufttur** [shopware/platform#8406]
### Shopware Version

6.7.0.0

### Affected area / extension

Extension:B2B-Suite

### Actual behaviour

Ensure all page content is contained by landmarks: for all navigation links and user name (9 issues)
**How to fix:** https://dequeuniversity.com/rules/axe/4.10/region?application=AxeChrome

![Image](https://github.com/user-attachments/assets/8a140eaa-c8f0-4fcb-b1e5-4fdc6a465d03)
![Image](https://github.com/user-attachments/assets/53384f40-b1bf-4029-962d-7f7a2d0da2bc)
![Image](https://github.com/user-attachments/assets/fab9a98e-2a03-4ee9-bc71-5d78b8e8d629)
![Image](https://github.com/user-attachments/assets/02bf2dd5-3063-4335-9173-93f25864207b)
![Image](https://github.com/user-attachments/assets/7b4e427b-eea8-40d4-9e92-1a201f267e58)
![Image](https://github.com/user-attachments/assets/3638e5f0-3b5e-48ec-ba2e-6e322b01ea49)
![Image](https://github.com/user-attachments/assets/00ede189-a35f-4482-8ebb-8d73f5d596b5)
![Image](https://github.com/user-attachments/assets/64d9f2ce-2bd7-465c-9e78-569741bdea35)
![Image](https://github.com/user-attachments/assets/e176751f-f633-4271-bffb-1b6ccbeb261d)

### Expected behaviour

All page content should be contained by landmarks

### How to reproduce

Run axe DevTools
